### PR TITLE
Add import tab UI for existing sites

### PR DESCRIPTION
**Pull request by sejas**
- Closes https://github.com/Automattic/dotcom-forge/issues/8157

## Proposed Changes

- Add UI to import by drag and drop a file.
- Add logic to pick a file from file system and import a site.
- Add tests to cover drag and drop hook and new import export tab.

## Testing Instructions

- Run `STUDIO_IMPORT_EXPORT=true npm start`
- Access an existing site
- Select Import / Export tab
- Click on the Import Drop zone and Select a valid file to import
- Observe the import works as expected displaying a confirmation modal before starting the import.
- Click on Start again
- Drag and drop a valid file to import
- Observe the import works as expected
- Do not click on Start again and Drag and drop a valid file to import
- Observe the import works as expected
- Repeat the steps with any other file that is not valid
- Observe a message of error

https://github.com/user-attachments/assets/ac07e062-d69c-448c-ae36-6cfdebb213cf

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

**Review by fluiddot — COMMENTED** [review]
Great work @sejas 🏅 !

After testing I spotted the following issue:

### No error message after importing an invalid file a second time

**Steps:**
1. Import an invalid file.
2. Observe that an error message is shown.
3. Click on Start again.
4. Import the same invalid file.
5. Observe no error message is shown.

**UPDATE:** Something similar happens when importing the same valid failed a second time. It doesn't execute the import process.

<hr>

Apart from the above, I'd like to share the following potential enhancements that shouldn't block the PR but would be interesting to consider:

### Add link icon following designs

I noticed there's a slight difference between these changes and the designs in the Open site button.

**Design:**
Screenshot 2024-07-22 at 11 13 18

**Current implementation:**
Screenshot 2024-07-22 at 11 13 27

### Allow dropping a new file after an import completed

Currently, after importing a site, we need to click on Starg again to drop a new file. If te user drags and drops a new file, I think we could omit this step and directly allow the new file to be imported.

#### Visual glitch when dropping a file

When dragging a file over the import drop target, I saw that on some areas the container changes its state:

https://github.com/user-attachments/assets/597c0513-7957-4694-86a1-c40d8ea655e3

### Adding a loading indicator when clicking on Open site if the site is not running

In some cases, like when using slow machines, starting a site could take some seconds. I wonder if we could add a loading indicator to let the user know that the site is loading, similar to what we have in the Start/Stop site button.

File: src/components/content-tab-import-export.tsx
Comment (fluiddot) [inline src/components/content-tab-import-export.tsx]: Nitpick: If the component we add to Learn more is a `Button`, for consistency, probably the HTML tag that we could use could be `button`.

Comment (fluiddot) [reply src/components/content-tab-import-export.tsx]: Nitpick: I noticed the button includes the dot, I presume the button should only contain the text Learn More, without any symbols.

```suggestion
						__( 'Import a Jetpack backup or a .sql database file. <a>Learn more</a>.' ),
```

File: src/components/content-tab-import-export.tsx
Comment (fluiddot) [inline src/components/content-tab-import-export.tsx]: Nitpick: We already have this URL defined in a constant, so we could use take from there.

https://github.com/Automattic/studio/blob/c8b1d1520ab8847c6c6b5fc9827a0e5e2cb54ef0/src/constants.ts#L13

File: src/ipc-types.d.ts
Comment (fluiddot) [inline src/ipc-types.d.ts]: Instead of defining it here again, what do you think about using the type exported in `lib/import-export/import/types.ts` in line 38 instead?

File: src/components/content-tab-import-export.tsx
Comment (sejas) [reply src/components/content-tab-import-export.tsx]: Changed to use button element, and moved the dot out: https://github.com/Automattic/studio/pull/369/commits/09cc3780fae2754fb249483c97e110e0ea61692b

File: src/components/content-tab-import-export.tsx
Comment (sejas) [reply src/components/content-tab-import-export.tsx]: Done! https://github.com/Automattic/studio/pull/369/commits/b9be0b1013bc7defef1d10bfafe3f68acab4a56f
We may want to point to another URL in the future.

File: src/ipc-types.d.ts
Comment (sejas) [reply src/ipc-types.d.ts]: That's something I tried first, but importing or exporting anything from/to `src/ipc-types.d.ts` results in a type error for the rest of types used in this file, I guess due to a cyclic dependency, and `npm run start` does not work anymore. Do you experience the same?
All types on `src/ipc-types.d.ts` are shared across the repository, but in this case I'm also using ImportState as a value.

Do you know any other way to achieve that? For now I've decided to remove the duplicated Enum and use the literals instead:
https://github.com/Automattic/studio/pull/369/commits/3a64bcca477f3bbadde88a8dc2ec9565ae610426

**Comment by sejas** [comment]
> I noticed there's a slight difference between these changes and the designs in the Open site button.

Done! https://github.com/Automattic/studio/pull/369/commits/4e818a4cc4bc1f9de23aa41689c394df62231213


> Adding a loading indicator when clicking on Open site if the site is not running
> In some cases, like when using slow machines, starting a site could take some seconds. I wonder if we could add a loading indicator to let the user know that the site is loading, similar to what we have in the Start/Stop site button.

Done! https://github.com/Automattic/studio/pull/369/commits/114f1d5404fba7b4746539382945317b47b8f668

https://github.com/user-attachments/assets/ed14e9fa-04c1-428e-8bd9-686a66727e66

I'll update later:
- No error message after importing an invalid file a second time
- Allow dropping a new file after an import completed
- Visual glitch when dropping a file

File: src/components/content-tab-import-export.tsx
Comment (katinthehatsite) [reply src/components/content-tab-import-export.tsx]: I think as per design, we should not need the dot at all:

Screenshot 2024-07-23 at 10 53 03 AM

**Comment by katinthehatsite** [comment]
I am not sure what I am testing wrong on my end but when I open the import tab, I am seeing this as my starting screen:

Screenshot 2024-07-23 at 10 51 41 AM

Instead of the screen where you can drag the file. What might be the reason for that?
I am trying to click on `Start again` but still nothing happens on my end.

Also, when I console.log `importState` for a given site, I keep on getting `undefined`.

File: src/components/content-tab-import-export.tsx
Comment (katinthehatsite) [inline src/components/content-tab-import-export.tsx]:
```suggestion
		checkboxLabel: __( 'Don't ask again' ),
```

File: src/hooks/use-drag-and-drop-file.ts
Comment (katinthehatsite) [inline src/hooks/use-drag-and-drop-file.ts]: Do we need this console.log?

File: src/hooks/use-drag-and-drop-file.ts
Comment (katinthehatsite) [inline src/hooks/use-drag-and-drop-file.ts]: Also, I am not sure, but does `onFileDrop` need to be added as a dependency here?

File: src/components/content-tab-import-export.tsx
Comment (sejas) [reply src/components/content-tab-import-export.tsx]: Fixed! https://github.com/Automattic/studio/pull/369/commits/59bee3582ca46cfdd082bbdcab6ac9e0500dbe67

File: src/hooks/use-drag-and-drop-file.ts
Comment (sejas) [reply src/hooks/use-drag-and-drop-file.ts]: You are right. Console logs are not needed, it was used just for debugging.

File: src/hooks/use-drag-and-drop-file.ts
Comment (sejas) [reply src/hooks/use-drag-and-drop-file.ts]: Yep!, I added it there 👌 . Good catch.

File: src/components/content-tab-import-export.tsx
Comment (sejas) [reply src/components/content-tab-import-export.tsx]: Done https://github.com/Automattic/studio/pull/369/commits/9cb451790240b22f9b974cab6c64a2b995987cdd

**Comment by sejas** [comment]
@fluiddot , @katinthehatsite , Thanks for reviewing the PR.

I've updated the code addressing your suggestions, added a screencast to the description, and added tests to cover `useDragAndDropFile` hook and `ContentTabImportExport` tab.

> Instead of the screen where you can drag the file. What might be the reason for that?
> I am trying to click on Start again but still nothing happens on my end.

I should have moved the state of the PR to in progress. The PR was in middle of the fixes. Now everything should work fine.

> Also, when I console.log importState for a given site, I keep on getting undefined.

That's expected, sites start with undefined importState until the user starts any import.


Let me know if you find anything else 👍

File: src/ipc-types.d.ts
Comment (fluiddot) [reply src/ipc-types.d.ts]: Yep, there are some caveats when using Enums ([reference](https://www.typescriptlang.org/docs/handbook/enums.html#const-enum-pitfalls)). In this case, I think we can solve it by using `declare` when defining the enum in `src/ipc-types.d.ts`. E.g.:

```patch
diff --git forkSrcPrefix/src/ipc-types.d.ts forkDstPrefix/src/ipc-types.d.ts
index 9b12bbc01a2689ab004b9a63d28c8998ea7e9469..b639575d51b7b95134477c6c8ac7cff38a9b2b5a 100644
--- forkSrcPrefix/src/ipc-types.d.ts
+++ forkDstPrefix/src/ipc-types.d.ts
@@ -11,6 +11,12 @@ interface ShowNotificationOptions extends Electron.NotificationConstructorOption
 	showIcon: boolean;
 }
 
+declare enum ImportState {
+	Initial = 0,
+	Importing = 1,
+	Imported = 2,
+}
+
 interface StoppedSiteDetails {
 	running: false;
 
@@ -29,7 +35,7 @@ interface StoppedSiteDetails {
 		supportsMenus: boolean;
 	};
 	isAddingSite?: boolean;
-	importState?: 0 | 1 | 2;
+	importState?: ImportState;
 }
 
 interface StartedSiteDetails extends StoppedSiteDetails {
diff --git forkSrcPrefix/src/lib/import-export/import/types.ts forkDstPrefix/src/lib/import-export/import/types.ts
index cc8b0b961a9e4ac75cb467ff958db1d3e21c2474..8687feba85f76f8a2ccc4dff28502c848bc8fd34 100644
--- forkSrcPrefix/src/lib/import-export/import/types.ts
+++ forkDstPrefix/src/lib/import-export/import/types.ts
@@ -23,10 +23,4 @@ export interface BackupArchiveInfo {
 	type: string;
 }
 
-export enum ImportState {
-	Initial = 0,
-	Importing = 1,
-	Imported = 2,
-}
-
 export type NewImporter = new ( backup: BackupContents ) => Importer;
diff --git forkSrcPrefix/src/components/content-tab-import-export.tsx forkDstPrefix/src/components/content-tab-import-export.tsx
index a97f0beb3a0700b7166977abd9bd3aa91629c25c..594dfe3bc686ba5f59c6bf41b2ebb7f38e07ac2e 100644
--- forkSrcPrefix/src/components/content-tab-import-export.tsx
+++ forkDstPrefix/src/components/content-tab-import-export.tsx
@@ -14,7 +14,6 @@ import { cx } from '../lib/cx';
 import { sanitizeFolderName } from '../lib/generate-site-name';
 import { getIpcApi } from '../lib/get-ipc-api';
 import { ExportOptions } from '../lib/import-export/export/types';
-import { ImportState } from '../lib/import-export/import/types';
 import Button from './button';
 import { ProgressBarWithAutoIncrement } from './progress-bar';
 
diff --git forkSrcPrefix/src/components/tests/content-tab-import-export.test.tsx forkDstPrefix/src/components/tests/content-tab-import-export.test.tsx
index d69ca6a1f30911732e18dbbe8cffae7a0a498f21..2988ef496c3ab93f8e8bb9007501cc5a03cc4b61 100644
--- forkSrcPrefix/src/components/tests/content-tab-import-export.test.tsx
+++ forkDstPrefix/src/components/tests/content-tab-import-export.test.tsx
@@ -2,7 +2,6 @@ import { render, fireEvent, waitFor, screen, createEvent, act } from '@testing-l
 import { userEvent } from '@testing-library/user-event';
 import { useSiteDetails } from '../../hooks/use-site-details';
 import { getIpcApi } from '../../lib/get-ipc-api';
-import { ImportState } from '../../lib/import-export/import/types';
 import { ContentTabImportExport } from '../content-tab-import-export';
 
 jest.mock( '../../lib/get-ipc-api' );
diff --git forkSrcPrefix/src/hooks/use-site-details.tsx forkDstPrefix/src/hooks/use-site-details.tsx
index 3a908384933900fce168840cbcb66d1a77aa5b5a..16062aa67d8d1eeb9b5a4a7938c02f7a541387aa 100644
--- forkSrcPrefix/src/hooks/use-site-details.tsx
+++ forkDstPrefix/src/hooks/use-site-details.tsx
@@ -10,7 +10,7 @@ import {
 	useState,
 } from 'react';
 import { getIpcApi } from '../lib/get-ipc-api';
-import { BackupArchiveInfo, ImportState } from '../lib/import-export/import/types';
+import { BackupArchiveInfo } from '../lib/import-export/import/types';
 import { sortSites } from '../lib/sort-sites';
 import { useSnapshots } from './use-snapshots';
 
```

Let me know if that works 🤞 .

Comment (fluiddot) [reply src/ipc-types.d.ts]: Although TypeScript doesn't complain, I noticed that this approach doesn't work in runtime 😞.

Comment (fluiddot) [reply src/ipc-types.d.ts]: I haven't found a better way than your original approach by duplicating. However, an alternative could be to stop using an enum and have string values defined in a type. Here's a potential approach:

```patch
diff --git forkSrcPrefix/src/components/tests/content-tab-import-export.test.tsx forkDstPrefix/src/components/tests/content-tab-import-export.test.tsx
index d69ca6a1f30911732e18dbbe8cffae7a0a498f21..7191147823793f1345ebb193559a6af8260151a8 100644
--- forkSrcPrefix/src/components/tests/content-tab-import-export.test.tsx
+++ forkDstPrefix/src/components/tests/content-tab-import-export.test.tsx
@@ -2,7 +2,6 @@ import { render, fireEvent, waitFor, screen, createEvent, act } from '@testing-l
 import { userEvent } from '@testing-library/user-event';
 import { useSiteDetails } from '../../hooks/use-site-details';
 import { getIpcApi } from '../../lib/get-ipc-api';
-import { ImportState } from '../../lib/import-export/import/types';
 import { ContentTabImportExport } from '../content-tab-import-export';
 
 jest.mock( '../../lib/get-ipc-api' );
@@ -15,7 +14,7 @@ const selectedSite: SiteDetails = {
 	path: '/test-site',
 	phpVersion: '8.0',
 	adminPassword: btoa( 'test-password' ),
-	importState: ImportState.Initial,
+	importState: 'initial',
 };
 
 describe( 'ContentTabImportExport Import', () => {
diff --git forkSrcPrefix/src/components/content-tab-import-export.tsx forkDstPrefix/src/components/content-tab-import-export.tsx
index a97f0beb3a0700b7166977abd9bd3aa91629c25c..1057db67b46eaf7817d84dd68970e737435fc19c 100644
--- forkSrcPrefix/src/components/content-tab-import-export.tsx
+++ forkDstPrefix/src/components/content-tab-import-export.tsx
@@ -14,7 +14,6 @@ import { cx } from '../lib/cx';
 import { sanitizeFolderName } from '../lib/generate-site-name';
 import { getIpcApi } from '../lib/get-ipc-api';
 import { ExportOptions } from '../lib/import-export/export/types';
-import { ImportState } from '../lib/import-export/import/types';
 import Button from './button';
 import { ProgressBarWithAutoIncrement } from './progress-bar';
 
@@ -174,15 +173,15 @@ const ImportSite = ( props: { selectedSite: SiteDetails } ) => {
 		}
 	};
 	const clearImportState = () => {
-		updateSite( { ...props.selectedSite, importState: ImportState.Initial } );
+		updateSite( { ...props.selectedSite, importState: 'initial' } );
 		clearImportFileInput();
 	};
 
 	const startLoadingCursorClassName =
 		loadingServer[ props.selectedSite.id ] && 'animate-pulse duration-100 cursor-wait';
 
-	const isImporting = props.selectedSite.importState === ImportState.Importing && ! isDraggingOver;
-	const isImported = props.selectedSite.importState === ImportState.Imported && ! isDraggingOver;
+	const isImporting = props.selectedSite.importState === 'importing' && ! isDraggingOver;
+	const isImported = props.selectedSite.importState === 'imported' && ! isDraggingOver;
 	const isInitial = ! isImporting && ! isImported;
 	return (
 		<div className={ cx( 'flex flex-col w-full', startLoadingCursorClassName ) }>
diff --git forkSrcPrefix/src/lib/import-export/import/types.ts forkDstPrefix/src/lib/import-export/import/types.ts
index cc8b0b961a9e4ac75cb467ff958db1d3e21c2474..8687feba85f76f8a2ccc4dff28502c848bc8fd34 100644
--- forkSrcPrefix/src/lib/import-export/import/types.ts
+++ forkDstPrefix/src/lib/import-export/import/types.ts
@@ -23,10 +23,4 @@ export interface BackupArchiveInfo {
 	type: string;
 }
 
-export enum ImportState {
-	Initial = 0,
-	Importing = 1,
-	Imported = 2,
-}
-
 export type NewImporter = new ( backup: BackupContents ) => Importer;
diff --git forkSrcPrefix/src/ipc-types.d.ts forkDstPrefix/src/ipc-types.d.ts
index 9b12bbc01a2689ab004b9a63d28c8998ea7e9469..eb1300fe137bb0f8e4ee0029b9f90524b09415a3 100644
--- forkSrcPrefix/src/ipc-types.d.ts
+++ forkDstPrefix/src/ipc-types.d.ts
@@ -11,6 +11,8 @@ interface ShowNotificationOptions extends Electron.NotificationConstructorOption
 	showIcon: boolean;
 }
 
+type ImportState = 'initial' | 'importing' | 'imported';
+
 interface StoppedSiteDetails {
 	running: false;
 
@@ -29,7 +31,7 @@ interface StoppedSiteDetails {
 		supportsMenus: boolean;
 	};
 	isAddingSite?: boolean;
-	importState?: 0 | 1 | 2;
+	importState?: ImportState;
 }
 
 interface StartedSiteDetails extends StoppedSiteDetails {
diff --git forkSrcPrefix/src/hooks/use-site-details.tsx forkDstPrefix/src/hooks/use-site-details.tsx
index 3a908384933900fce168840cbcb66d1a77aa5b5a..b56304c087fc5ba9bd6565765a93eca4be2b5a0a 100644
--- forkSrcPrefix/src/hooks/use-site-details.tsx
+++ forkDstPrefix/src/hooks/use-site-details.tsx
@@ -10,7 +10,7 @@ import {
 	useState,
 } from 'react';
 import { getIpcApi } from '../lib/get-ipc-api';
-import { BackupArchiveInfo, ImportState } from '../lib/import-export/import/types';
+import { BackupArchiveInfo } from '../lib/import-export/import/types';
 import { sortSites } from '../lib/sort-sites';
 import { useSnapshots } from './use-snapshots';
 
@@ -227,11 +227,11 @@ export function SiteDetailsProvider( { children }: SiteDetailsProviderProps ) {
 	);
 
 	const importFile = useCallback( async ( file: BackupArchiveInfo, selectedSite: SiteDetails ) => {
-		let finalImportState = ImportState.Imported;
+		let finalImportState: ImportState = 'imported';
 		try {
 			setData( ( prevSites ) =>
 				prevSites.map( ( site ) =>
-					site.id === selectedSite.id ? { ...site, importState: ImportState.Importing } : site
+					site.id === selectedSite.id ? { ...site, importState: 'importing' } : site
 				)
 			);
 			const backupFile: BackupArchiveInfo = {
@@ -243,7 +243,7 @@ export function SiteDetailsProvider( { children }: SiteDetailsProviderProps ) {
 				title: selectedSite.name,
 				body: __( 'Import complete' ),
 			} );
-			finalImportState = ImportState.Imported;
+			finalImportState = 'imported';
 		} catch ( error ) {
 			getIpcApi().showMessageBox( {
 				type: 'error',
@@ -253,7 +253,7 @@ export function SiteDetailsProvider( { children }: SiteDetailsProviderProps ) {
 				),
 				buttons: [ __( 'OK' ) ],
 			} );
-			finalImportState = ImportState.Initial;
+			finalImportState = 'initial';
 		} finally {
 			setData( ( prevSites ) =>
 				prevSites.map( ( site ) =>

```

Comment (fluiddot) [reply src/ipc-types.d.ts]: I don't have a strong preference over the two and the current approach, so feel free to pick any option.

File: src/components/content-tab-import-export.tsx
Comment (fluiddot) [inline src/components/content-tab-import-export.tsx]: I'm curious why we need to use the original component `Button` instead of the customized one 🤔. I've tried reverting this change and the buttons look the same.

File: src/components/content-tab-import-export.tsx
Comment (fluiddot) [inline src/components/content-tab-import-export.tsx]: By adding a fake delay in the import process, I noticed that we allow importing a file while another import is going on. Probably this is caused by this condition, as `isDraggingOver` will make `isImporting` false.

Unless we want to explicitly allow interrupting an import, which I presume it's not the case, we could try removing `&& ! isDraggingOver` from the condition.

https://github.com/user-attachments/assets/3a724dd3-39bf-48c5-836d-c6dde8e381ee

File: src/components/content-tab-import-export.tsx
Comment (fluiddot) [inline src/components/content-tab-import-export.tsx]: Following the above comment, we shouldn't probably display the border while dragging to avoid confusion about the behavior.

File: src/components/content-tab-import-export.tsx
Comment (fluiddot) [inline src/components/content-tab-import-export.tsx:236]: I wonder if the intent here was to use the same icon we have for the other Open site button, instead of using a different symbol 🤔 .

Screenshot 2024-07-23 at 18 32 02


E.g.:

```patch
diff --git forkSrcPrefix/src/components/content-tab-import-export.tsx forkDstPrefix/src/components/content-tab-import-export.tsx
index a97f0beb3a0700b7166977abd9bd3aa91629c25c..688f603099529fbe16808fc91402b1243986f788 100644
--- forkSrcPrefix/src/components/content-tab-import-export.tsx
+++ forkDstPrefix/src/components/content-tab-import-export.tsx
@@ -2,7 +2,7 @@ import { speak } from '@wordpress/a11y';
 import { Button as CoreButton } from '@wordpress/components';
 import { createInterpolateElement } from '@wordpress/element';
 import { sprintf, __ } from '@wordpress/i18n';
-import { Icon, download } from '@wordpress/icons';
+import { Icon, download, external } from '@wordpress/icons';
 import { useI18n } from '@wordpress/react-i18n';
 import { format } from 'date-fns';
 import { useRef } from 'react';
@@ -226,7 +226,12 @@ const ImportSite = ( props: { selectedSite: SiteDetails } ) => {
 										variant="primary"
 										onClick={ openSite }
 									>
-										{ __( 'Open site ↗' ) }
+										{ __( 'Open site' ) }
+										<Icon
+											className="ltr:ml-0.5 rtl:mr-0.5 rtl:scale-x-[-1]"
+											icon={ external }
+											size={ 14 }
+										/>
 									</Button>
 									<Button variant="link" className="!px-2.5 !py-2" onClick={ clearImportState }>
 										{ __( 'Start again' ) }

```

File: src/ipc-types.d.ts
Comment (sejas) [reply src/ipc-types.d.ts]: Thanks for trying different alternatives, I think using string literals is a better approach. I've applied the changes here https://github.com/Automattic/studio/pull/369/commits/eca0bdb6d313e04a328b728af09e263e7ae68289
I've also removed the 'initial' state because that can be identified when the state is different from importing or imported and by default is undefined.

File: src/hooks/tests/use-drag-and-drop-file.test.tsx
Comment (fluiddot) [inline src/hooks/tests/use-drag-and-drop-file.test.tsx]: I presume there's a type here:

```suggestion
	test( 'should set isDraggingOver to false on drag leave event', () => {
```

**Review by fluiddot — COMMENTED** [review]
Thank you so much @sejas for addressing all the issues I shared in [the review](https://github.com/Automattic/studio/pull/369#pullrequestreview-2191108645). I've tested the PR again and looks great to me.

I added some comments, but the one I think it would be great to solve before merging is [the one related to interrupting the import process when dropping another file](https://github.com/Automattic/studio/pull/369#discussion_r1688355462). With that addressed, the PR will be good for to be merged 🎊 .

File: src/components/content-tab-import-export.tsx
Comment (sejas) [reply src/components/content-tab-import-export.tsx:236]: @matt-west , what icon should we use?

Current design and implemented:
Screenshot 2024-07-23 at 19 23 25

External icon for consistency
image

File: src/components/content-tab-import-export.tsx
Comment (sejas) [reply src/components/content-tab-import-export.tsx]: Changed ! https://github.com/Automattic/studio/pull/369/commits/877e6aa799f35c2039ad7bab79cb0e7139168fbf
I used CoreButton because that was the button introduced on https://github.com/Automattic/studio/pull/365/files#diff-fd133fa5ff9b8ef225c2775f90a5122225974fff6288e81b6fd518ef9bbb2e0bR1

File: src/components/content-tab-import-export.tsx
Comment (sejas) [reply src/components/content-tab-import-export.tsx]: Great catch! 
Changed https://github.com/Automattic/studio/pull/369/commits/9153649af06c7b98ae6fe0041d50e4d6798dc051

File: src/hooks/tests/use-drag-and-drop-file.test.tsx
Comment (sejas) [reply src/hooks/tests/use-drag-and-drop-file.test.tsx]: Done https://github.com/Automattic/studio/pull/369/commits/d71d91da2273140f854bb79661db6324d7e1f913

File: src/components/content-tab-import-export.tsx
Comment (sejas) [reply src/components/content-tab-import-export.tsx]: Changed https://github.com/Automattic/studio/pull/369/commits/9153649af06c7b98ae6fe0041d50e4d6798dc051#diff-fd133fa5ff9b8ef225c2775f90a5122225974fff6288e81b6fd518ef9bbb2e0bR212

File: src/components/content-tab-import-export.tsx
Comment (fluiddot) [reply src/components/content-tab-import-export.tsx:236]: @sejas Since the feature is behind a feature flag, we could merge the PR as is and in another PR update the icon if necessary based on Matt's response. WDTY?

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 ! Awesome work @sejas 🏅 !

**Pull request merged**
